docker/stand-alone/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+services:
+  sam-app:
+    image: broadinstitute/sam:stand-alone
```

Comment:
Before merging, we will need to update this to use the correct image.  (Is sam:dev the right image?)